assets/hmr.js

```diff
@@ -74,14 +80,7 @@ async function applyUpdate(id) {
   return true;
 }
 
-const socket = new WebSocket('ws://localhost:12321/');
-
-socket.addEventListener('open', function (event) {
```

Comment:
I still find these two useful for debugging. Lets convert to use the `debug()` function for now, and then as an enhancement we can add some logic to turn the debug function on/off (maybe via a `window.HMR_DEBUG_MODE` flag)